server/routes/api.js

```diff
@@ -17,6 +27,44 @@ router.get("/", (_, res) => {
 		}
 	});
 });
+
+// GET all the Users.
+router.get("/users", (_, response) => {
+	pool.query("SELECT * FROM user", (err, result) => {
+		if (err) {
+			logger.error(err);
+			response.status(500).json({ message: err.message });
+		} else {
+			response.json(result.rows);
+		}
+	});
+});
+
+// GET Return a single User by ID.
+router.get("/users/:userId", function (request, response) {
+	const userId = Number(request.params.userId);
+
+	if (Number.isNaN(userId) || !Number.isSafeInteger(userId) || userId <= 0) {
+		genericIntErrorMessage(userId, response, 1);
+		return;
+	}
+
+	pool.query("SELECT * FROM users WHERE id=$1", [userId], (err, result) => {
+		if (err) {
+			logger.error(err);
+			response.status(500).json({ message: err.message });
+		} else {
+			let reply = result.rows;
+			if (reply.length === 0) {
+				genericIntErrorMessage(userId, response, 2, "User");
+				return;
+			}
+
+			return response.json(reply); // Success
+		}
+	});
+});
```

Comment:
Please note the users endpoint has already been implemented and we get users from auth0. I'm not sure why you are working on users endpoints